packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -786,6 +788,20 @@ const getGridStyles = (
 
     border: 'none',
 
+    // add a box shadow on hover and selection for all body cells
+    '& :not(.rdg-summary-row, .rdg-header-row) .rdg-cell': {
```

Comment:
this feels like it would be an expensive selector, since it implies checking every element inside the grid. i was thinking maybe this-ish:

```
'.rdg-row-even, .rdg-row-odd' {
  '> .rdg-cell': {
  }
}
```

but the summary row currently gets an even/odd class :(, we should propose an upstream fix.

maybe for now we can make it somewhat cheaper by only targeting children and not all descendant nodes.

```
> :not(.rdg-summary-row, .rdg-header-row) > .rdg-cell'
```

probably wont make a huge difference now, but will with HTML/markdown cells, or bargauge/lcd cells i think it will have a positive effect.